data/raw/keybindings.json

```
@@ -588,6 +588,13 @@
     "name": "Toggle armor visibility on character sprite",
     "bindings": [ { "input_method": "keyboard_any", "key": "H" } ]
   },
+  {
+    "type": "keybinding",
+    "id": "CHANGE_SPRITE",
+    "category": "SORT_ARMOR",
+    "name": "Change sprite of armor.",
+    "bindings": [ { "input_method": "keyboard_any", "key": "C" } ]
```

I don't think this should be available as a "top-level" keybinding. Check out #59677 for the reasoning.

I think it should be a key found within the status menu (@ screen)

---

In addition, this (and `SORT_ARMOR` above) should be changed to specify the modifier key for the key code mode like this:

```suggestion
    "bindings": [ { "input_method": "keyboard_char", "key": "C" }, { "input_method": "keyboard_code", "key": "c", "mod": [ 
"shift" ] } ]
```

---

moved to status menu.